Error Thrown:
Traceback (most recent call last):
  File "c:\Users\cbodine\miniforge3\envs\ping\Lib\site-packages\pingmapper\gui_main.py", line 720, in gui
    shutil.rmtree(projDir)
  File "c:\Users\cbodine\miniforge3\envs\ping\Lib\shutil.py", line 781, in rmtree
    return _rmtree_unsafe(path, onexc)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\cbodine\miniforge3\envs\ping\Lib\shutil.py", line 635, in _rmtree_unsafe
    onexc(os.unlink, fullname, err)
  File "c:\Users\cbodine\miniforge3\envs\ping\Lib\shutil.py", line 633, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Access is denied: '.\\.git\\objects\\01\\34f0288f12bb3343e91fdbc25c8a32b6883225'
